README.md

```diff
@@ -1,63 +1,74 @@
-the current release, it's intended for advanced home users, DIY NAS builders
-and smaller organizations that need NAS or private cloud in a box. Go to
-[rockstor.com](http://rockstor.com) for more information.
+Rockstor aims to aid individuals and organisations alike. As a DIY setup some
+familiarity with PC/Pi4 operating system install is required; but not much beyond
+the basics. For more information please see our main web site:
+[rockstor.com](http://rockstor.com)
 
-# Getting started
+# Getting started & any questions
 
-Best way to get started is by following the [quickstart
+The best way to get started is by following the [quickstart
 guide](http://rockstor.com/docs/quickstart.html).
 
-To get in touch with developers, users and contributors join our
-[community forum](http://forum.rockstor.com).
+To get in touch with developers, users, and contributors join our
+[community forum](http://forum.rockstor.com) and ask away.
 
 # Documentation
 
-Documentation is in the
-[rockstor-doc](https://github.com/rockstor/rockstor-doc) repository. You can also read it
-[online](http://rockstor.com/docs).
+[Our Documentation](http://rockstor.com/docs) is also a community endeavour with it's
+own GitHub repo: [rockstor-doc](https://github.com/rockstor/rockstor-doc): contributions
+and correction are always welcome.
 
 # Troubleshooting
 
-The [community forum](http://forum.rockstor.com) is the ideal place to look for
-help and ask other users and developers questions.
+The [community forum](http://forum.rockstor.com) is the ideal location for help and 
+support; all current developers are also active forum members.
 
 # Issue tracking
 
-You can go through all open issues for rockstor-core
-[here](https://github.com/rockstor/rockstor-core/issues).
+We use GitHub's issue system, although a forum post/discussion is advised prior to
+opening a GitHub issue as this often helps in identifying exactly what should be added
+to an new issue.
+[Rocksor issues](https://github.com/rockstor/rockstor-core/issues).
 
 # License
 
 Please see the file [COPYING](https://github.com/rockstor/rockstor-core/blob/master/COPYING).
 
 # Contributing
 
-Development environment setup and contribution guidelines are explained
-[here](http://rockstor.com/docs/contribute.html).
+Development environment setup and contribution guidelines are available in our docs:
+[Contributing to Rockstor - Overview](http://rockstor.com/docs/contribute.html)
+
+There is no contributor licence agreement.
 
-Pull requests are welcome. Pick an issue that interests you or create a new one
-to work on. You can also ask on the [community forum](http://forum.rockstor.com) for
-guidance. We'll be happy to help.
+Pull requests are welcome. Pick an issue that interests you or create a new one.
+Please also references our [friendly community forum ](http://forum.rockstor.com)
+for open questions and our ever growing wiki entries that serve as our
```

Comment:
```suggestion
for open questions and our ever-growing wiki entries that serve as our
```

Reply:
Agreed. @FroggyFlox over time and many reviews has warmed to to the use of the hyphen for readability.